app/views/tags/index.html.erb

```diff
@@ -21,8 +21,23 @@
 
 <main id="main-content" class="crayons-layout crayons-layout--header-inside">
   <header class="crayons-page-header">
-    <h1 class="crayons-title"><%= t("views.tags.heading") %></h1>
-    <%= link_to t("views.tags.following_tags"), dashboard_following_tags_path, class: "crayons-btn crayons-btn--ghost" %>
+    <h1 class="crayons-title">
+      <% if params.key?(:q) && params[:q].present? %>
```

Comment:
Suggestion: same as above.

```suggestion
      <% if params[:q].present? %>
```

Reply:
thanks for these suggestions! I feel like I've been bitten in the past by not explicitly checking if the key was in the `params`. maybe that was way back in some older version of rails/ruby or a wonky codebase :she

Reply:
You definitely can get bitten by that but generally not when there's a presence check, as they are somewhat redundant. It definitely can happen with hashes and boolean values though, since `false` is not `present?`. 

As you said, things changed a lot over the years, we have the safe-navigation operator now (`&.`), etc. so this should be good 😃